addon/components/o-s-s/marketing-banner.ts

```diff
@@ -0,0 +1,48 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { assert } from '@ember/debug';
+
+type Options = {
+  primaryGradiantColor?: string;
+  secondaryGradiantColor?: string;
+  backgroundGridHidden?: boolean;
```

Comment:
I think @phndiaye prefers "positive" arguments, how about we go with `backgroundGridDisplayed` or `backgroundGridShown` ?

Reply:
+ you already have a `backgroundGridDisplay` that could do the job :)